hypnotoad/gui/gui.py

```diff
@@ -323,8 +324,22 @@ def run(self):
             )
             return
 
+        # Call read_geqdsk to recreate self.eq object in case any settings needed in
+        # __init__ have been changed
+        self.read_geqdsk()
+
         self.statusbar.showMessage("Running...")
-        self.mesh = BoutMesh(self.eq)
+        try:
+            self.mesh = BoutMesh(self.eq)
+        except (ValueError, SolutionError):
+            self.statusbar.showMessage(
+                "Error in grid generation, change settings and run again!"
+            )
```

Comment:
What if we instead pop up a message box with the exception message? I can do that quick if you like

Reply:
That would be great! :+1:

Reply:
Thanks @ZedThree!